### ISSUE TITLE
skeleton example from official docs throwing a UnicodeEncodeError

### ISSUE THREAD

**Issue by palewire** [conda/conda-build#1089]
I followed all the instructions from the newbie tutorial, installing miniconda on Linux and building up to this:

```bash
$ conda skeleton pypi pyinstrument
```

Here's what happened:

```
Traceback (most recent call last):
  File "/home/ben/miniconda2/bin/conda-skeleton", line 5, in <module>
    sys.exit(main())
  File "/home/ben/miniconda2/lib/python2.7/site-packages/conda_build/main_skeleton.py", line 300, in main
    args_func(args, p)
  File "/home/ben/miniconda2/lib/python2.7/site-packages/conda_build/main_build.py", line 389, in args_func
    args.func(args, p)
  File "/home/ben/miniconda2/lib/python2.7/site-packages/conda_build/main_skeleton.py", line 315, in execute
    pypi.main(args, parser)
  File "/home/ben/miniconda2/lib/python2.7/site-packages/conda_build/pypi.py", line 338, in main
    is_url)
  File "/home/ben/miniconda2/lib/python2.7/site-packages/conda_build/pypi.py", line 380, in get_download_data
    data = client.release_data(package, version) if not is_url else None
  File "/home/ben/miniconda2/lib/python2.7/xmlrpclib.py", line 1243, in __call__
    return self.__send(self.__name, args)
  File "/home/ben/miniconda2/lib/python2.7/xmlrpclib.py", line 1602, in __request
    verbose=self.__verbose
  File "/home/ben/miniconda2/lib/python2.7/site-packages/conda_build/pypi.py", line 233, in request
    return self.parse_response(resp)
  File "/home/ben/miniconda2/lib/python2.7/site-packages/conda_build/pypi.py", line 240, in parse_response
    p.feed(resp.text)
  File "/home/ben/miniconda2/lib/python2.7/xmlrpclib.py", line 558, in feed
    self._parser.Parse(data, 0)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xd7' in position 3552: ordinal not in range(128)
```

**Comment by msarahan** [comment]
Thanks.  I'll investigate this for the next bugfix release.

**Comment by palewire** [comment]
Thanks. In the meantime, should I try building my first package from
scratch?

**Comment by msarahan** [comment]
Sure - there's plenty of examples out there, if you'd like to copy one and build on it.  Many of the recipes that are used to create Anaconda are at https://github.com/continuumIO/anaconda-recipes

There's also conda-forge, which is a community collection of recipes.  These use the Jinja2 templating feature of conda-build more, which might raise the learning curve a bit, but they're still good examples.  https://github.com/conda-forge/feedstocks.

**Comment by msarahan** [comment]
Fixed by #1092

**Comment by msarahan** [comment]
This should be fixed in the 1.21.4 release.  Closing this issue - please re-open it if problems recur.
